src/main/java/org/util/string/StringUtil.java

```diff
@@ -270,10 +283,11 @@ public char get(int index) {
      * @arg int index, char value
      * @returns void
      */
-    // TODO: setCharacter
+
     public StringUtil set(int index, char value) {
-        builder.toString().toCharArray()[index] = value;
-        return new StringUtil(builder);
+        StringUtil other = new StringUtil();
```

Comment:
not correct, setting one character is similar to replacing only 1 character with other value